### PR TITLE
Fixes grammar issues on descriptions (Part 2)

### PR DESCRIPTION
**Pull request by Ktlwjec**
Title.

### CODE REVIEW

File: code/game/objects/items/storage/backpack.dm [hunk 1]
```diff
@@ -68,7 +68,7 @@
 
 /obj/item/storage/backpack/santabag
 	name = "Santa's Gift Bag"
-	desc = "Space Santa uses this to deliver presents to all the nice children in space in Christmas! Wow, it's pretty big!"
+	desc = "Space Santa uses this to deliver presents to all the nice children in space at Christmas! Wow, it's pretty big!"
```
Comment (Amelia0010) [inline code/game/objects/items/storage/backpack.dm]: I'd say `for christmas` tbh

File: code/modules/clothing/suits/miscellaneous.dm [hunk 2]
```diff
@@ -461,7 +461,7 @@
 
 /obj/item/clothing/suit/gothcoat
 	name = "gothic coat"
-	desc = "Perfect for those who want stalk in a corner of a bar."
+	desc = "Perfect for those who want to stalk in a corner of the bar."
```
Comment (nichlas0010) [inline code/modules/clothing/suits/miscellaneous.dm]: mason time

File: code/modules/food_and_drinks/food/snacks_other.dm [hunk 3]
```diff
@@ -139,7 +139,7 @@
 
 /obj/item/reagent_containers/food/snacks/carrotfries
 	name = "carrot fries"
-	desc = "Tasty fries from fresh Carrots."
+	desc = "Tasty fries from fresh carrots."
```
Comment (Amelia0010) [inline code/modules/food_and_drinks/food/snacks_other.dm:142]: clearly this is actually referring to your pig not an actual carrot, so the fix would be to capitalize Carrot fries

File: code/game/objects/items/storage/backpack.dm (hunk 1 above)
Comment (Amelia0010) [reply code/game/objects/items/storage/backpack.dm]: or just `to deliver christmas presents to all the nice children in space`, flows better IMO

**Comment by HomocidlePancakes** [comment]
Sec magboots description is still scuffed

**Review by alexkar598 — APPROVED** [review]

**Pull request merged**